docs/Onco-Query-Language.md

```diff
@@ -134,6 +162,71 @@ Or over-expressed at the phospho-protein level:
 EGFR_PY992: PROT > 2
 ```
 
+<br>
+
+<a name="modifiers"></a>
+## Modifiers
+Modifiers can be used on their own or in combination with other OQL terms for mutations and copy number alterations to further refine the query. Modifiers can be combined with other OQL terms using an underscore.
+
+<a name="driver"></a>
+### Driver
+The `DRIVER` modifier applies to mutations, fusions and copy number alterations. The definition of what qualifies as a driver alteration comes from the "Mutation Color" menu in OncoPrint. By default, drivers are defined as mutations, fusions and copy number alterations in <a href="http://oncokb.org">OncoKB</a> or <a href="http://www.cancerhotspots.org">CancerHotspots</a>.
+
+On its own, the `DRIVER` modifier includes driver mutations, fusions and copy number alterations:
+```
+EGFR: DRIVER
+```
+
+Or it can be used in combination with another OQL term. For example, to see only driver fusion events:
+```
+EGFR: FUSION_DRIVER
+```
+
+Or driver missense mutations:
+```
+EGFR: MUT = MISSENSE_DRIVER
+```
+
+When combining `DRIVER` with another OQL term, the order doesn't matter: `MUT_DRIVER` and `DRIVER_MUT` are equivalent. `DRIVER` can be combined with:
+* `MUT`
+* `MUT = <TYPE>` or `MUT = <protein change>`
+* `FUSION`
+* `CNA`
+* `AMP` or `GAIN` or `HETLOSS` or `HOMDEL`
+* `GERMLINE` or `SOMATIC` (see below)
+
+
+<a name="germline-somatic"></a>
+### Germline/Somatic
+The `GERMLINE` and `SOMATIC` modifiers only apply to mutations. A mutation can be explicitly defined as germline during the data curation process. Note that very few studies on the public cBioPortal contain germline data.
+
+`GERMLINE` or `SOMATIC` can be combined with:
+* `MUT`
+* `MUT = <TYPE>` or `MUT = <protein change>`
+* `DRIVER`
+
+To see all germline BRCA1 mutations:
+```
+BRCA1: GERMLINE
+```
+
+Or to see specifically truncating germline mutations:
+```
+BRCA1: TRUNC_GERMLINE
+```
+
+Or to see somatic missense mutations:
+```
+BRCA1: MUT = MISSENSE_SOMATIC
+``` 
+
+When combining `GERMLINE` or `SOMATIC` with `DRIVER` and another term (e.g. `NONSENSE`), it is essential that the middle term is `GERMLINE` or `SOMATIC` or `DRIVER`; `GERMLINE_NONSENSE_DRIVER` will not work.
```

Comment:
I find it confusing to explain this in terms of the middle term... would it be clearer to say "the mutation term must appear at the beginning or the end"?

Reply:
@adamabeshouse is it possible to support any order?

Reply:
yes, ill look into that

Reply:
@tmazor we're now supporting any order so this line isn't necessary.

Reply:
Great - I updated the page accordingly @adamabeshouse